lib/perl/Genome/Sample/Command/Import/Base.t

```diff
@@ -4,10 +4,28 @@ use strict;
 use warnings;
 
 use above 'Genome';
-use Test::More tests => 1;
+use Test::Exception;
+use Test::More;
 
-# This test was auto-generated because './Sample/Command/Import/Base.pm'
-# had no '.t' file beside it.  Please remove this test if you believe it was
-# created unnecessarily.  This is a bare minimum test that just compiles Perl
-# and the UR class.
-use_ok('Genome::Sample::Command::Import::Base');
+use_ok('Genome::Sample::Command::Import::Base') or die;
+class SampleImporter {
+    is => 'Genome::Sample::Command::Import::Base',
+    has => {
+        nomenclature => {},
+        # these are deprecated
+        name_regexp => { default_value => '(TGI\-[\w\d]+)\-[\w\d]+', },
+    },
+};
+
+my $taxon = Genome::Taxon->__define__(name => 'almost human');
+ok($taxon, 'defined taxon');
```

Comment:
It is. It is/was/going to be included in another PR to be used further in the test. I will remove it.